pages/profile/redeem/[code].tsx

```diff
@@ -103,16 +103,15 @@ const ReedemCode: FC = () => {
   }
 
   useEffect(() => {
-    if (contextLoaded && user && router && router.query.code) {
+    if (contextLoaded && user && router.query.code && !router.query.inputCode) {
       redeemingCode(router.query.code);
     }
   }, [user, contextLoaded, router.query.code]);
 
   const changeRouteCode = () => {
     router.push(`/profile/redeem/${inputCode}?inputCode=${false}`);
 
-    const codeFromUrl = router.query.code;
-    redeemingCode(codeFromUrl);
+    redeemingCode(inputCode);
   };
 
   return ready && user ? (
```

Comment:
After cancelling the error `inputCode` value remains `false`, resulting into a blank screen

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/37466979/235838594-5dc72091-c145-4a55-8265-16829a9d9709.png">

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/37466979/235838704-0ecb7a97-7fe7-42d9-97b9-46a436d544bb.png">

Reply:
> After cancelling the error `inputCode` value remains `false`, resulting into a blank screen
> 
> <img alt="image" width="1435" src="https://user-images.githubusercontent.com/37466979/235838594-5dc72091-c145-4a55-8265-16829a9d9709.png">
> 
> <img alt="image" width="1435" src="https://user-images.githubusercontent.com/37466979/235838704-0ecb7a97-7fe7-42d9-97b9-46a436d544bb.png">

@mariahosfeld what should I do in this case ?

Reply:
> After cancelling the error `inputCode` value remains `false`, resulting into a blank screen
> 
> <img alt="image" width="1435" src="https://user-images.githubusercontent.com/37466979/235838594-5dc72091-c145-4a55-8265-16829a9d9709.png">
> 
> <img alt="image" width="1435" src="https://user-images.githubusercontent.com/37466979/235838704-0ecb7a97-7fe7-42d9-97b9-46a436d544bb.png">

Even not clicking to error  cancel button, the model disappear after some time , I'm not sure, Is this  a default behaviour or need to do some changes in it